src/Interpreters/ActionsVisitor.cpp

```diff
@@ -1109,6 +1110,12 @@ void ActionsMatcher::visit(const ASTFunction & node, const ASTPtr & ast, Data &
         }
     }
 
+    /// Normal functions are not parametric for now.
```

Comment:
This `if` is not correct because executable UDF can have parameters. See #38142, #37929 and #30957.